index.ts

```diff
@@ -33,6 +34,9 @@ async function main() {
         // get devpool issues
         const devpoolIssues: Issue[] = await getAllIssues(DEVPOOL_OWNER_NAME, DEVPOOL_REPO_NAME);
 
+        //to assign issues to assignee when they have been taken
+        assignAssigneeToUnavailableIssues(devpoolIssues);
```

Comment:
@pavlovcik assigning bounty hunters to issues from some unknown repo seems to be a strange UX, besides there will be unnecessary notifications. Adding the `Unavailable` label would be a better solution. What do you think?